src/main/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiService.scala

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.dsde.firecloud.webservice
+
+import org.broadinstitute.dsde.firecloud.model.Trial.TrialOperations
+import org.broadinstitute.dsde.firecloud.model.Trial.TrialOperations._
+import org.broadinstitute.dsde.firecloud.model.UserInfo
+import org.broadinstitute.dsde.firecloud.service.TrialService.TrialServiceMessage
+import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectives, PerRequestCreator, TrialService}
+import org.broadinstitute.dsde.firecloud.utils.StandardUserInfoDirectives
+import spray.httpx.SprayJsonSupport
+import spray.json.DefaultJsonProtocol._
+import spray.routing.{HttpService, Route}
+
+trait TrialApiService extends HttpService with PerRequestCreator with FireCloudDirectives
+  with StandardUserInfoDirectives with SprayJsonSupport {
+
+  private implicit val executionContext = actorRefFactory.dispatcher
+  val trialServiceConstructor: () => TrialService
+
+  val trialApiServiceRoutes: Route = {
+    post {
+      path("trial" / "manager" / "enable|disable|terminate".r) { (operation: String) =>
```

Comment:
Very minor, probably out of scope for this PR, and likely defined in a spec doc elsewhere, but I am curious why we aren't using HTTP verbs here, specifically `DELETE` for `terminate`. Arguably, we could be using `PUT` for disable.